dk-installer.py

```diff
@@ -1854,4 +2015,13 @@ def execute(self, args):
             TestgenDeleteDemoAction(),
         ],
     )
-    sys.exit(installer_instance.run())
+    installer_instance.run()
+
+
+if __name__ == "__main__":
+    arguments = sys.argv[1:]
+    print("DataKitchen Installer")
+    if len(arguments) == 0:
+        show_main_menu()
```

Comment:
Let's start with showing the menu only for the windows .exe. We don't want the current installation experience being affected by the changes.

Reply:
Then, won't this PR be ever merged? How are we planning to keep this "special version"?

Reply:
I mean shouldn't there be a condition to check whether we are using windows or not? And only if that condition is true, the menu is displayed. Otherwise, the default is how things currently work.

Reply:
I made a function that checks if the code is being run in .py or .exe. If it's .exe the menu will be shown.